k8s-operator/apis/v1alpha1/types_tsdnsconfig.go

```diff
@@ -6,6 +6,7 @@
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit for consistency/readability at the point of usage:

```suggestion
	corev1 "k8s.io/api/core/v1"
```